sources/Afup/Comptabilite/Comptabilite.php

```diff
@@ -171,7 +171,16 @@ function obtenirJournal($debitCredit = '',
         $requete .= 'compta_reglement.reglement, ';
         $requete .= 'compta_evenement.evenement, ';
         $requete .= 'compta_categorie.categorie, ';
-        $requete .= 'compta_compte.nom_compte    ';
+        $requete .= 'compta_compte.nom_compte,    ';
+        $requete .= '(compta.montant_ht_soumis_tva_0 + compta.montant_ht_soumis_tva_5_5 + compta.montant_ht_soumis_tva_10 + compta.montant_ht_soumis_tva_20) as montant_ht,   ';
+        $requete .= '((compta.montant_ht_soumis_tva_5_5*0.055) + (compta.montant_ht_soumis_tva_10*0.1) + compta.montant_ht_soumis_tva_20*0.2) as montant_tva,   ';
```

Comment:
Comme @stakovicz je pense qu'il faut les parenthèse ici pour garantir le calcul et améliorer la lisibilité 
```suggestion
        $requete .= '((compta.montant_ht_soumis_tva_5_5*0.055) + (compta.montant_ht_soumis_tva_10*0.1) + (compta.montant_ht_soumis_tva_20*0.2)) as montant_tva,   ';
```